src/main/java/de/telran/projectredirectservice/service/RedirectService.java

```diff
@@ -0,0 +1,25 @@
+package de.telran.projectredirectservice.service;
+
+import com.google.common.hash.Hashing;
+import org.springframework.stereotype.Service;
+
+import java.nio.charset.StandardCharsets;
+
+@Service
+public class RedirectService {
+
+    public String create(String longUrl) {
+        String hash_id = Hashing.murmur3_128()
+                .hashString( longUrl, StandardCharsets.UTF_8 )
+                .toString();
+
```

Comment:
We may need to have some logic around expiration date. Let's discuss business rules together.